pkgs/default.nix

```diff
@@ -3,31 +3,19 @@
 with pkgs.lib;
 
 let
-  mkModule = path: pkgs.callPackage ./moduleit/entrypoint.nix {
-    configPath = path;
+  modules = import ./modules {
```

Comment:
this is importing modules in a second place in addition to the top level outputs.modules. I'd prefer to pass `self` into the pkgs evaluation and get the modules from `self.modules`